oauthproxy.go

```diff
@@ -633,7 +633,30 @@ func getRemoteAddr(req *http.Request) (s string) {
 	return
 }
 
+// See https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/http-caching?hl=en
+var noCacheHeaders = map[string]string{
+	"Expires":         time.Unix(0, 0).Format(time.RFC1123),
+	"Cache-Control":   "no-cache, no-store, must-revalidate, max-age=0",
```

Comment:
it seems like the only header needed should be `Cache-Control: no-store` and this is confirmed by https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control

Reply:
Your point is correct.
In reality, however, many servers set multiple values to cache-control to prevent caching.

I've run the chrome dev tool on the login screen of many web services (google, microsoft, etc.) to check the headers.
Many of them were `cache-control: no-cache, no-store, max-age=0, must-revalidate`.
Why is that?

Reply:
"copy-paste and it seems to always work so far" is as likely as any other explanation. But fair enough, it'll work.